MontyHallv2/DoorCreation/Door.cs

```diff
@@ -0,0 +1,42 @@
+namespace MontyHallv2.DoorCreation;
+
+public class Door
+{
+    private bool _opened;
```

Comment:
```suggestion
    private bool _isOpen;
```
nit pick, but if you think about a door object (you're doing OOP, where instances of theses classes represent an object i.e. an actual door in this case), you can open it and close it more than once.

if open the door, then `_opened` should be set to true. pretty simple. but now if i close the door... should `_opened` be true or false? I could argue that the door has been opened before (as in at least once).

again, a contrived example, but less ambiguity for little effort is great!